### PR TITLE
Implement `annotate` and `in_bulk` (updated)

### PR DESCRIPTION
**Pull request by phitranphitranphitran**
All I did was fix the flake8 errors in https://github.com/stphivos/django-mock-queries/pull/135

### CODE REVIEW

**Review by aguevaracaprio — APPROVED** [review]

**Comment by stphivos** [comment]
hi @phitranphitranphitran CI only fails with py2.7. I will create a new version `2.2` and drop support for python 2, so we can merge this after.

**Comment by stefan6419846** [comment]
Are you still interested in this PR? Could you rebase your branch on the latest *master* in this case, please?

File: django_mock_queries/utils.py [hunk 1]
```diff
@@ -1,6 +1,12 @@
 from datetime import datetime, date
 from django.core.exceptions import FieldError
-from unittest.mock import Mock
```
Comment (stefan6419846) [inline django_mock_queries/utils.py:3]: The `from unittest.mock import Mock` line should stay as on the *master* branch.

**Comment by stefan6419846** [comment]
I am not sure about the *codeclimate* stuff though, let's wait for @stphivos to comment on this.

**Comment by stphivos** [comment]
Codeclimate stuff are not a required check, not sure it still needs force merge. About the codecove failure it's these 2 lines:
```python
except ImportError:
    from mock import Mock
```
Reverting to just `from unittest.mock import Mock` should pass.

File: django_mock_queries/utils.py (hunk 1 above)
Comment (phitranphitranphitran) [reply django_mock_queries/utils.py:3]: Done

**Pull request merged**